dim/benches/movie_scanner.rs

```diff
@@ -123,15 +121,15 @@ fn movie_scanner_bench(c: &mut Criterion) {
             .unwrap();
 
         b.to_async(rt).iter_custom(|iters| async move {
-            let path = env!("CARGO_TARGET_TMPDIR").to_string();
+            let _path = env!("CARGO_TARGET_TMPDIR").to_string();
             let mut db = bootstrap().await;
             let mut total_elapsed = Duration::ZERO;
 
             for _ in 0..iters {
                 let library_id = create_library(&mut db).await;
                 let start = Instant::now();
 
-                scan_directory(&mut db, library_id, vec![path.clone()]).await;
+                // scan_directory(&mut db, library_id, vec![path.clone()]).await;
```

Comment:
que???

Reply:
it was failing to compile, will fix later